app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/launcher/LauncherActivity.kt

```diff
@@ -0,0 +1,50 @@
+package org.mifos.mobile.cn.ui.mifos.launcher
+
+import android.content.Intent
+import android.os.Bundle
+import android.os.Handler
+import org.mifos.mobile.cn.R
+import org.mifos.mobile.cn.ui.base.MifosBaseActivity
+import org.mifos.mobile.cn.ui.mifos.DashboardActivity
+import org.mifos.mobile.cn.ui.utils.ConstantKeys
+import javax.inject.Inject
+
+/**
+ * @author Manish Kumar
+ * On 31/05/18.
+ */
+
+class LauncherActivity:MifosBaseActivity(),LauncherContract.View {
+
+
+    @Inject
+    lateinit var launcherPresenter: LauncherPresenter
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_launcher)
```

Comment:
why are you using this method, it's not compatible for all the devices, you have to make layout for all the different devices to enable the support.  See the blogs that I shared with you and make this launch activity using drawable layout as mentioned in blog that is the best way to support all the devices.

Reply:
@therajanmaurya I didn't get the message you sent me can you send again, please. 
Do you want me to use a launcher theme and change the ```windowBackground``` into custom drawable .. Plus aren't we going to use the splash activity to reroute to different pages?